### PR TITLE
some ui things

### PR DESCRIPTION
**Pull request by srajankumar**

### CODE REVIEW

File: web/client/src/app/hackathon/page.tsx [hunk 1]
```diff
@@ -15,14 +15,18 @@ export default async function HackathonPage() {
...
-      <H1 className="-ml-1">Welcome Leader {user.data.name}!</H1>
-      <H2 className="mt-4 font-light">Register your Team for the Hackathon</H2>
-      <div>
+    <main className="container max-w-5xl mx-auto px-8 lg:px-20 xl:px-24 py-12">
+      <h2 className="mt-3 tracking-wide text-3xl font-bold">
+        Welcome, Leader {user.data.name}!
+      </h2>
+      <h3 className="text-2xl mt-4 font-light">
+        Register your Team for the Hackathon
+      </h3>
```
Comment (dev-shetty) [inline web/client/src/app/hackathon/page.tsx:26]: There is H1 and H2 component, you can override styles on that

**Review by dev-shetty — CHANGES_REQUESTED** [review]
I have added `.prettier.rc` file just make sure you again save the edited files so the semincolons and all which were added now get removed.

Also add the screenshot of the pages UI you changed will help a lot.

**Comment by srajankumar** [comment]
image

image

image

image

image

**Comment by srajankumar** [comment]
snapshots of the changes

**Comment by dev-shetty** [comment]
@srajankumar this space seems too much on PC (Above the title Codeblaze, even in Dashboard I fell)

image

Maybe it looks good on mobile but it is just empty space with no use, try to reduce it

**Comment by srajankumar** [comment]
> @srajankumar this space seems too much on PC (Above the title Codeblaze, even in Dashboard I fell)
> 
> image
> 
> Maybe it looks good on mobile but it is just empty space with no use, try to reduce it

Decreased padding 

image

**Review by dev-shetty — APPROVED** [review]
LGTM ✅

**Pull request merged**